src/Evolution/DiscontinuousGalerkin/Actions/InternalMortarDataImpl.hpp

```diff
@@ -173,8 +173,7 @@ void internal_mortar_data(
       // optimization. This will require making the
       // Normalized<UnnormalizedFaceNormal> a simple tag instead of a
       // compute tag.
-      db::mutate<
-          evolution::dg::Tags::InternalFace::NormalCovectorAndMagnitude<Dim>>(
+      db::mutate<evolution::dg::Tags::NormalCovectorAndMagnitude<Dim>>(
```

Comment:
This commit "Add normal covector quantities to external boundaries" appears to be 95% changing the namespace for some Tags, and 5% behavior changes. Please split it into two commits.

Reply:
Without the functionality changes the previous namespace was "more correct" It's the functionality changes that is triggering the rename, that's why they are in the same commit (and why I think they belong in the same commit). Thoughts?

Reply:
I'd be okay leaving the commit as-is if you improve the documentation of it by changing the commit message so (in the lengthy part) it has two bullets... one bullet explaining the functionality change and one bullet explaining the namespace change. How does that sound?